readme.md

```diff
@@ -51,14 +51,29 @@ composer require fzaninotto/faker
 
 ## Basic Usage
 
-Use `Faker\Factory::create()` to create and initialize a faker generator, which can generate data by accessing properties named after the type of data you want.
+### Autoloading
+
+Faker supports both `PSR-0` as `PSR-4` autoloaders. 
```

Comment:
Not sure if users care about the kind of autoloader we use internally.

Perhaps we should point out two options:

- when installed via `composer`, require `vendor/autoload.php`
- otherwise, require `/path/to/Faker/src/autoload.php`

What do you think?

Reply:
I do agree but this is a bit "as is" and i think we should not just wipe it. The focus now is on composer autoload and otherwise the custom one which is psr0.

But i would not be suprised if there are enough people around that just load it manually instead and that do not use composer yet.